src/main/java/com/nha/abdm/wrapper/hrp/mongo/tables/Logs.java

```diff
@@ -0,0 +1,110 @@
+package com.nha.abdm.wrapper.hrp.mongo.tables;
+
+import org.springframework.data.mongodb.core.mapping.Document;
+import org.springframework.data.mongodb.core.mapping.Field;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@Document(collection = "logs")
+public class Logs {
```

Comment:
lets call this class as `RequestLog` and collection as `request-logs`

Reply:
Done re-naming.